Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -181,6 +181,8 @@ public virtual bool CanLaunchWebBrowser
 
         private IPlugin[] _plugins;
 
+        private IEnumerable<LocalPlugin> _pluginsManifests;
```

Comment:
```suggestion
        private IReadOnlyList<LocalPlugin> _pluginsManifests;
```

No reason to keep this as `IEnumerable`